ui/src/components/Dashboard/index.jsx

```diff
@@ -219,30 +215,25 @@ export default class Dashboard extends Component {
       staticContext: _,
       ...props
     } = this.props;
-    const { error, navOpen, showHelpView, showLogo } = this.state;
+    const { error, navOpen, showHelpView } = this.state;
     const drawer = (
       <nav>
         <div className={classes.toolbar}>
+          <img
+            className={classes.logoStyle}
+            height={30}
+            alt="logo"
+            src={Logo}
+          />
           <Typography
-            onMouseEnter={this.handleTitleToggle}
-            onMouseLeave={this.handleTitleToggle}
             {...!process.env.DOCS_ONLY && {
               component: Link,
               to: '/',
```

Comment:
Clicking on the image will will not redirect the user to `/` anymore. I think we can have this fixed by doing something like (untested):

```jsx
<Link className={classes.toolbar}>
  <img ... />
  <Typography ... />
</Link>
```

Reply:
I tried pushing my changes to this pull-request but for some reason it didn't work. I created https://github.com/taskcluster/taskcluster/pull/620 with the additional changes. Your contributions are still recorded in the PR :)